process_jartic_simple.py

```
@@ -0,0 +1,163 @@
+                                                continue
+                                            
+                                            out_file.write(
+                                                f"{timestamp.isoformat()},JARTIC_{location_code},{location_name},"
+                                                f"0,0,traffic_volume,{traffic_volume},vehicles/5min,{prefecture}\n"
+                                            )
+                                            
+                                            measurements_in_archive += 1
+                                            update_progress()
+                                                
+                                        except Exception:
+                                            continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)